auth/src/main/res/layout/fui_phone_layout.xml

```diff
@@ -1,53 +1,70 @@
-            android:layout_width="match_parent"
-            android:layout_height="wrap_content"
-            android:layout_marginTop="@dimen/fui_field_padding_vert"
-            android:textColor="?android:textColorTertiary"
-            android:textIsSelectable="false"
-            app:layout_constraintTop_toBottomOf="@+id/send_code" />
-
-    </android.support.constraint.ConstraintLayout>
-
-</ScrollView>
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <android.support.constraint.ConstraintLayout
+            style="@style/FirebaseUI.WrapperStyle"
+            android:layout_height="wrap_content">
+
+            <com.firebase.ui.auth.ui.phone.CountryListSpinner
+                android:id="@+id/country_list"
+                style="@style/FirebaseUI.CountrySpinner"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintBaseline_toBaselineOf="@+id/phone_layout" />
+
+            <com.firebase.ui.auth.util.ui.BaselineTextInputLayout
+                android:id="@+id/phone_layout"
+                style="@style/FirebaseUI.TextInputLayout.PhoneField"
+                android:layout_width="0dp"
+                android:layout_height="wrap_content"
+                app:layout_constraintStart_toEndOf="@+id/country_list"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintTop_toTopOf="parent">
+
+                <android.support.design.widget.TextInputEditText
+                    android:id="@+id/phone_number"
+                    style="@style/FirebaseUI.TextInputEditText.PhoneField"
+                    android:imeOptions="actionDone" />
+
+            </com.firebase.ui.auth.util.ui.BaselineTextInputLayout>
+
+            <Button
+                android:id="@+id/send_code"
+                style="@style/FirebaseUI.VerifyPhoneButton"
+                android:layout_marginTop="@dimen/fui_field_padding_vert"
+                android:text="@string/fui_verify_phone_number"
+                app:layout_constraintTop_toBottomOf="@+id/phone_layout" />
+
+            <TextView
+                android:id="@+id/send_sms_tos"
+                style="@style/FirebaseUI.Text.BodyText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/fui_field_padding_vert"
+                android:textColor="?android:textColorTertiary"
+                android:textIsSelectable="false"
+                app:layout_constraintTop_toBottomOf="@+id/send_code" />
+
+        </android.support.constraint.ConstraintLayout>
+
+    </ScrollView>
+
+    <TextView
+        android:id="@+id/email_footer_tos_and_pp_text"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/fui_field_padding_vert"
+        android:paddingRight="15dp"
```

Comment:
nit: 15 is a strange number, try 16 which is a standard Android padding amount (normally you work in multiples of 4dp)